plugins/scaffolder-actions/scaffolder-backend-argocd/src/new-backend.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2024 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
can you remove this file entirely, and just export the scaffolderBackendArgoCD from the index? In otherwords, move this line 16 to the index file.

Reply:
@vanessap-aa ah got it. Done. Thank you :)